client/jetpack-connect/signup.js

```diff
@@ -119,11 +120,16 @@ export class JetpackSignup extends Component {
 		this.props.resetAuthAccountType();
 	};
 
-	isWoo() {
+	isWooOnboarding() {
```

Comment:
As a person without much knowledge of Woo will find it confusing between this and `isWooCoreProfiler`. I think we give it a name with a signal that it's an older version (or deprecated). Maybe, `isWooDeprecatedOnboarding`?

Reply:
I agree, but I was thinking of changing this after we release the core profiler because it has not been deprecated yet. And we also need to deprecate all `isJetpackWooCommerceFlow `. What do you think?

Reply:
That makes sense, let's deprecate everything on a follow-up! 👍